File: self-study/Go/core/2/1.go
package main

import (
	"fmt"

)

// Даны три действительных числа. Найти максимальное и минимальное из них.

func main() {
	var a, b, c float64

	fmt.Print("Enter three float numbers: ")
	fmt.Scan(&a, &b, &c)

	max := max(a, b, c)
	min := min(a, b, c)

	fmt.Printf("Max: %.2f, Min: %.2f", max, min)
}

File: academic/Python/2/1.py
# 1. Даны действительные числа x,y. Получить: max(x,y), min(x,y).

a, b = map(float, input("Введите два числа: ").split())

max_ = max(a, b)
min_ = min(a, b)

print(f"Минимальное: {min_}; Максимальное: {max_}")
